### PR TITLE
Italian Translation

### PR DESCRIPTION
**Pull request by wsx79**

### CODE REVIEW

File: resources/language/resource.language.it_it/strings.po [hunk 1]
```diff
@@ -0,0 +1,2259 @@
...
+"Report-Msgid-Bugs-To: alanwww1@xbmc.org\n"
+"POT-Creation-Date: 2014-10-26 17:05+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: it\n"
+"Tranlsation: wsx79
```
Comment (jurialmunkey) [inline resources/language/resource.language.it_it/strings.po]: Missing newline and quote marks  ` \n"` at end of line here. Also, should be `Translation` not `Tranlsation`

**Review by wsx79 — COMMENTED** [review]
Fix done

**Comment by jurialmunkey** [comment]
Excellent! Thanks 😃

**Comment by drinfernoo** [comment]
@jurialmunkey Can I get a version bump on this?

**Pull request merged**